recipes/rgi/meta.yaml

```diff
@@ -1,29 +1,43 @@
 {% set name = "rgi" %}
 {% set version = "4.2.2" %}
-{% set sha256 = "ab4a5614e359884129064ec318d8dda2516052b26aa38cfd480245d24e0c96e7" %}
```

Comment:
Could you actually keep this variable set up here at the top?  You can also add another variable `sha256database` for the database.